src/layout/RemoveTagIcon.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+let RemoveTagIcon = props => (
+  <span className="tags-input-tag-remove" {...props}>
```

Comment:
since this is only used by the default theme, I think maybe this should just be part of the default tag component and not be on the theme